subjects/regressions/reko-90/PP.c

```diff
@@ -10452,22 +10460,58 @@ byte fn0800_8F7F(byte al, selector ds, segptr32 ptrArg02)
 
 word16 fn0800_9764(byte dl, selector ds, segptr32 ptrArg02, word16 wArg04, ptr16 & dlOut, ptr16 & diOut, ptr16 & dsOut)
 {
-	fn0800_8B95((byte) (fp - 0x06), dl, DPB(ptrLoc12, fp - 0x06, 0));
-	fn0800_8BA8((byte) (fp - 0x0A), (byte) ss, DPB(ptrLoc12, fp - 0x0A, 0));
-	word16 dx_48;
-	word16 di_49;
-	selector ds_50;
-	word16 ax_51 = fn0800_C04F(ds, DPB(ptrLoc16, fp - 0x06, 0), DPB(ptrLoc12, fp - 0x0A, 0), out dx_48, out di_49, out ds_50);
-	if ((wArg02 | wArg04) != 0x00)
-	{
-		word16 bx_80 = (word16) ptrArg02;
-		selector es_81 = SLICE(ptrArg02, selector, 16);
-		Mem84[es_81:bx_80 + 0x02:word16] = dx_48;
-		Mem85[es_81:bx_80 + 0x00:word16] = ax_51;
-	}
-	byte dl_69;
-	*dlOut = (byte) dx_48;
-	return ax_51;
+fn0800_9764_entry:
+	def fp
```

Comment:
Another regression. I have a feeling that all these regressions are related and will likely be fixed by a single change.